.github/actions/back/deploy/action.yml

```diff
@@ -47,12 +47,20 @@ runs:
         service: ${{ env.SERVICE_NAME }}
         image: ${{ steps.variables.outputs.image }}:latest
         region: us-central1
+        flags: |
+          --service-account=cloud-run@${{ env.PROJECT_ID }}.iam.gserviceaccount.com
+          --max-instances=10
+          --min-instances=${{ env.MIN_INSTANCES }}
+          --set-cloudsql-instances=app
```

Comment:
**thought**: i thought this needed to be in a different format. Did you test this?